packages/@okta/vuepress-theme-prose/global-components/AnnouncementBanner.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div v-if="bannerIsOpened" class="announcement-banner">
+    <div v-on:click="handleBannerClose" class="announcement-banner--close">
+      <i v-on:click="handleBannerClose" class="fa fa-times"></i>
+    </div>
+    <div class="announcement-banner--content">
+      <span class="announcement-banner--title">
+        {{ title }}
+      </span>
+      <div class="announcement-banner--description-wrapper">
+        <p class="announcement-banner--description">
+          <slot name="description"></slot>
+        </p>
+        <a
+          v-if="ctaHref && ctaText"
+          :href="ctaHref"
+          class="announcement-banner--cta"
+        >
+          {{ ctaText }} &rsaquo;
```

Comment:
Should the right arrow always be included? Would it be better to just leave it to the `ctaText` to specify this?

Reply:
That particular button always has it in the mocks but I don't mind moving it out.